lib/servo.js

```diff
@@ -275,8 +274,8 @@ Servo.prototype.to = function(degrees, time, rate) {
   if (this.invert) {
     degrees = Board.map(
       degrees,
-      this.range[0], this.range[1],
-      this.range[1], this.range[0]
+      0, 180,
+      180, 0
```

Comment:
If we invert based on user range:
```
var servo = new Five.Servo({
  range: [20, 140],
  invert: true
});
servo.to(20); // Moves servo to 40
```
If the values in range are not equidistant from the center position of the servo it will not be mapped correctly.